qualtran/bloqs/factoring/mod_add.py

```diff
@@ -48,9 +48,9 @@ class CtrlScaleModAdd(Bloq):
     def signature(self) -> 'Signature':
         return Signature(
             [
-                Register('ctrl', bitsize=1),
-                Register('x', bitsize=self.bitsize),
-                Register('y', bitsize=self.bitsize),
+                Register('ctrl', bitsize=QBit()),
+                Register('x', bitsize=QAny(self.bitsize)),
+                Register('y', bitsize=QAny(self.bitsize)),
```

Comment:
how should I be reviewing this PR? I would expect this to be `QInt` unless this PR is only part of the migration

Reply:
No this is actually a mistake. Are these unsigned or signed integers?

Reply:
Unsigned I think right?

Reply:
yes